src/averageAges.js

```diff
@@ -15,11 +15,14 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(x => x.sex === 'm');
+  const menFilter = century
+    ? men.filter(x => (Math.ceil(x.died / 100) === +century))
+    : men;
+  const totalAges = (prev, x) => prev + (x.died - x.born);
```

Comment:
1. If you create a function its name should start with verb (action). This one should be smth like `getTotalAges`
2. It is better to write this function inside reduce method because it is not used anywhere else.